.github/workflows/lint-pr-title.yml

```diff
@@ -0,0 +1,21 @@
+name: "Lint PR title"
+
+on:
+  pull_request_target:
```

Comment:
By using `pull_request_target` instead of `pull_request` it'll allow the workflow to run on forked PRs as well. As a downside, it will only run the action once it's on main (meaning not for this PR).

Reply:
@theyokohamalife You can trick github into registering an action. But in this case I do not think that is necessary. It loos good to me.